website/content/docs/configuration/seal/seal-ha.mdx

```diff
@@ -0,0 +1,76 @@
+---
+layout: docs
+page_title: Seal High Availability (Beta) - Seals - Configuration
+description: |-
+  How to configure multiple Seals for high availability.
+---
+
+# Seal High Availability (Beta)
+
+<Note title="Seal HA functionality requires Vault Enterprise">
+
+  Seal High Availability requires Vault Enterprise.
+
+</Note>
+
+-> **Warning:** This feature is available as a Beta for evaluation and should not 
+be used in production deployments of Vault.
+
+[Seal High Availability](/vault/docs/content/concepts/seal#Seal-High-Availability) 
+is the ability to configure more than one seal in order to have resilience against
+outage of a seal service or mechanism.   
+
+Using Seal HA involves configuring extra seals in Vault's server configuration file
+and restarting Vault, after having enabled the Seal HA beta feature by setting
+the environment variable:
+
+`VAULT_ENABLE_SEAL_HA_BETA=true`
+
+Before using Seal HA, one must have upgraded to Vault 1.15 or higher, and started
+Vault without the feature.  Afterward, the feature may be enabled by adding
+a seal
+
+## Adding and Removing Seals
+
+Seal HA adds two new fields to the seal configuration stanza, `name`, and
+`priority`:
+
+```hcl
+seal [TYPE] {
+  name = "seal-name"
+  priority = "1"
+  # ...
+}
+```
+
+Name is optional, and if not specified is set to the type of the seal.  Names
+must be unique.  If using two seals of the same type name must be specified.
+Internally, name is used to disambiguate seal wrapped values in some cases,
+so renaming seals should be avoided if possible.
+
+Priority is mandatory if more than one seal is specified.  Priority tells Vault
+the order in which to try seals during unseal (least priority first), 
+in the case more than one seal can unwrap a seal wrapped value, the order
+in which to attempt decryption, and which order to attempt to source entropy
+for entropy augmentation.  This can be useful if your seals have different
+performance or cost characteristics.
+
+To add an additional seal, simply add another seal stanza, specifying priority
+and optionally name, and restart Vault.
+
+To remove a seal, remove the corresponding seal stanza and restart.  There must
```

Comment:
I think it is not mandatory for having at least one seal remaining. If all are deleted, then it is going to be defaulted to shamir seal right?

Reply:
Yes, but at that point it won't be able to unseal or unwrap seal wrapped values, right?

Reply:
Yes, you are right. Thanks.